lib/chef/knife/ec_base.rb

```diff
@@ -119,7 +121,7 @@ def user_acl_rest
                            elsif server.supports_user_acls?
                              rest
                            elsif server.direct_account_access?
-                             Chef::REST.new("http://127.0.0.1:9465")
+                             Chef::ServerAPI.new("http://127.0.0.1:9465", {:api_version => "0"})
```

Comment:
This should be a no-op impact as this would be talking to old opscode-account which doesn't understand the version headers at all and which doesn't exist on systems that do.  However, it also shouldn't hurt.